app/views/layouts/_breadcrumbs.html.erb

```diff
@@ -3,7 +3,11 @@
     <div class="breadcrumbs">
       <%= link_to('Home', root_path) %>
       &raquo;
-      <%= link_to('Thesis Submission', new_thesis_path) %>
+        <% if current_page?(new_thesis_path) %>
+          Thesis Submission
+        <% else %>
+          <%= link_to('Thesis Submission', new_thesis_path) %>
```

Comment:
My understanding is that breadcrumbs are supposed to show where users are in the navigational hierarchy (so if people aren't on the thesis page, this shouldn't be a link to it, but rather an indication of where they are now).

Do we actually need breadcrumbs at all on a site with only like 3 pages, though, come to think of it? All of which are represented in the nav menu?

Reply:
I'm not sure we actually have any other obvious navigation, but definitely agree this is a non-traditional (and backwards) use of breadcrumbs. I'll open a ticket to have UX weigh in on the general navigation and call this out specifically.

Reply:
I'm going to remove the breadcrumbs and add pill nav.

Reply:
++ on removing breadcrumbs (we don't need them in every app just because we have the css, right?)

We have the buttons in the local app header. Given that there are only 3 pages on the whole site I think they cover it.

Reply:
There's no obvious way for a user on the home page to get to the submission page. I'll add that and then we're good.